lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../player.rb'
+require 'pry'
+
+class Human < Player
+	attr_accessor :name
+	def move(arr)
+		puts "What move would you like to make?"
+		move = gets.chomp
+		move
+	end

```

Comment:
Because of Ruby's implicit return, the varible `move` is unnecessary in this method. It could be simplified to
```
def move(arr)
  puts "What move would you like to make?"
  move = gets.chomp
end
```
But since `move` isn't being used elsewhere in this method, it could be further simplified to 
```
def move(arr)
  puts "What move would you like to make?"
  gets.chomp
end
```